apim-operator/pkg/controller/api/constants.go

```diff
@@ -1,25 +1,26 @@
 package api
 
 const (
-	dockerConfig = "docker-config"
+	dockerConfig  = "docker-config"
```

Comment:
Why all of the diff in the file is adding unwanted extra spaces? Is this a go formatting standard?
```diff
--dockerConfig  = "docker-config"
++dockerConfig = "docker-config"

Reply:
This is how it added with go formatting